scripts/postinstall.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p assets
```

Comment:
You don't need this line since the assets dir will be created by the next line with the -p option.